maintainers/maintainer-list.nix

```diff
@@ -7891,6 +7891,12 @@
     githubId = 3621083;
     name = "Roosembert (Roosemberth) Palacios";
   };
+  rovanion = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add rovanion``.